toolkit/scripts/toolchain/cgmanifest.json

```diff
@@ -5,11 +5,11 @@
                 "Type": "DockerImage",
                 "DockerImage": {
                     "Name": "mcr.microsoft.com/cbl-mariner/base/core",
-                    "Digest": "sha256:ca9eb22782f247189c15a4145dfde7f21bca25e545a6a3782a3fe2dbe3b128f0",
-                    "Tag": "2.0.20231004"
+                    "Digest": "sha256:82314abb594a695fd8817774e8b7f101934902cc1d99b3075e80acbc8b9b23ee",
```

Comment:
This one also looks like it's from a different PR.

Reply:
yea, I wanted to unblock this branch so i could run a test build